.travis.yml

```diff
@@ -56,6 +56,8 @@ jobs:
       - fastlane deploy flavor:surveillance_covid19_kenya
       - fastlane deploy flavor:vhw_burundi
     - script:
+        - fastlane deploy flavor:msf_goma
```

Comment:
indenting is off here
you might consider moving it to line 49 to reuse one of the existing jobs

Reply:
Addressed.